### PR TITLE
API Renames

### PR DESCRIPTION
**Pull request by IanMatthewHuff**
For #6014 

-   [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [ ] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Comment by IanMatthewHuff** [comment]
As usual for this will push at EOD.

File: src/client/common/application/notebook.ts
Comment (IanMatthewHuff) [inline src/client/common/application/notebook.ts:98]: Trying to remember to update variable names as I do renames like this, just to keep the naming clearer when types change.

**Review by rchiodo — APPROVED** [review]

**Comment by IanMatthewHuff** [comment]
Bleh, looks like the proposed API that I copied from notebook/dev actually has type errors in it. I believe that I'll need to wait until notebook/dev merge to main and then update this PR again.

**Comment by IanMatthewHuff** [comment]
The proposed API had some stuff in the test section that were types in the non-proposed API, but in the 1.57 types that were not published yet. Rather than mess with our build I just remove the Test class specific types from our proposed API copy.

**Comment by IanMatthewHuff** [comment]
Breaking changes so CI failures are expected. Merging.

**Pull request merged**
